h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+import sys
+import traceback
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import os
+from h2o.exceptions import H2OResponseError
+
+
+def test_parser_svmlight_column_skip_not_supported():
+    print("Test that functions calling fail if skipped_columns is passed with svm file.")
+    # generate a big frame with all datatypes and save it to svmlight
+    nrow = 10
+    ncol = 10
+    seed = 12345
+
+    f1 = h2o.create_frame(rows=nrow, cols=ncol, real_fraction=0.5, integer_fraction=0.5, missing_fraction=0.2,
+                          has_response=False, seed=seed)
+    f2 = h2o.create_frame(rows=nrow, cols=1, real_fraction=1, integer_fraction=0, missing_fraction=0,
+                          has_response=False, seed=seed)
+    f1.set_name(0, "target")
+    f1 = f2.cbind(f1)
+
+    tmpdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath('__file__')), "..", "results"))
+    if not (os.path.isdir(tmpdir)):
+        os.mkdir(tmpdir)
+    savefilenamewithpath = os.path.join(tmpdir, 'out.svm')
+    pyunit_utils.write_H2OFrame_2_SVMLight(savefilenamewithpath, f1)  # write h2o frame to svm format
+
+    try:
+        print("Test upload SVM file")
+        h2o.upload_file(savefilenamewithpath, skipped_columns=[5])
+        assert False, "Test should have thrown an exception due skipped_columns parameter is present"  # should have failed here
+    except H2OResponseError:
+        print("Test OK, finished with H2OResponseError")
+        pass
+    except Exception as e:
+        assert False, "Test finish with unexpected exception." + str(e)
```

Comment:
Do we need to catch the exception at all? Is this adding anything?

Reply:
The idea here is to be sure that it test throws only ResponseError. If it throws e. g. TypeError, ValueError then something is wrong in my opinion.

Reply:
I dont think you need the `except Exception` branch

Reply:
All right, I will remove it.